CHANGELOG.md

```diff
@@ -10,15 +10,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+[v0.11.0][v0.11.0_log]
+--------------
+
 ### Added
-- OE SDK Release packages can now be built on non-SGX and non-FLC machines.
+- Open Enclave SDK release packages can now be built on non-SGX and non-FLC machines.
 - Support for arbitrarily large thread-local data for SGX machines.
+- Experimental support for OpenSSL inside enclaves has been added while building the SDK from source.
+     - Use BUILD_OPENSSL flag while compiling the SDK.
+     - [OpenSSLSupport.md](docs/OpenSSLSupport.md) documents supported options and configuration needed to use OpenSSL inside an enclave.
+- Custom claims buffer serialization/de-serialization helper functions
+oe_verify_evidence() and oe_verify_attestation_certificate_with_evidence() have been added to the host-verify package.
```

Comment:
This is a bullet by itself:

**-** oe_verify_evidence() and oe_verify_attestation_certificate_with_evidence() have been added to the host-verify package.